### ISSUE TITLE
Code highlighting not appearing for Live Queries

### ISSUE THREAD

**Issue by thecandlestick** [silverbulletmd/silverbullet#634]
Example of a page with syntax highlighting (below) and how it appears when queried (above). Version 0.5.11

![2024-01-17-204853_782x550_scrot](https://github.com/silverbulletmd/silverbullet/assets/44912084/c2ce7d11-e684-45a1-bdcc-e243fe0ec708)
![2024-01-17-204931_776x511_scrot](https://github.com/silverbulletmd/silverbullet/assets/44912084/b961b0fd-dc09-43c0-9704-096914c2a828)

**Comment by zefhemel** [comment]
Right. This may seem surprising, but could be a bit harder to fix. Will put it on the list to look into.
